zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/LicenseController.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller;
+
+import io.camunda.service.ManagementServices;
+import io.camunda.zeebe.gateway.rest.ConditionalOnRestGatewayEnabled;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@ConditionalOnRestGatewayEnabled
+@RestController
+@RequestMapping(path = {"/v1", "/v2"})
```

Comment:
❓ Why we adding this to both versions? 🤔 Is this expected? Leaving this to @houssain-barouni I think he is more into the REST API topic.

Reply:
**Note**: The controller is removed from this PR here. It will come with the next PR related to https://github.com/camunda/camunda/issues/20540.

Nonetheless, some implementation hints:
* Please only add it to V2. We should not backport this to V1 of the REST API. New endpoints and features should only be added to V2.
* Please use the `@CamundaRestController` annotation instead of `@ConditionalOnRestGatewayEnabled` and `@RestController`. If we are to change annotations for all REST controllers, we can do that centrally using `@CamundaRestController`.
* @houssain-barouni, only the C8 REST Query API (search endpoints) are not working by default with 8.6., i.e. the controllers annotated with `CamundaRestQueryController`.